src/avvoenv/source/vault.rs

```diff
@@ -75,6 +80,14 @@ impl Client {
         Ok(())
     }
 
+    pub fn app_id_auth(&mut self, app_id: String, user_id: String) -> Result<(), errors::Error> {
+        self.resolve_leader()?;
+        let request = AuthAppIdRequest { user_id: user_id };
```

Comment:
You should be able to shorten the struct initialisation to `AuthAppIdRequest { user_id }` (it's a newer syntax feature, so that's why you might have spotted other places that repeat the attribute/variable name).